tests/test_beebop.py

```diff
@@ -9,3 +10,8 @@ def test_version():
 def test_add():
     assert addition.add(5, 4) == 9
     assert addition.add(5, 10) == 15
+
+
+def test_get_version():
+    assert versions.get_version(["beebop"]) =="[{\"name\": \"beebop\", \"version\": \"0.1.0\"}]"
```

Comment:
This is fine (taking into account the suggestions above about the return format and the argument) but we also would like some integration tests.

From this SO answer (https://stackoverflow.com/a/45665619) it looks like there's a test client we could/should use here https://flask.palletsprojects.com/en/1.1.x/testing/#the-first-test

Reply:
I have implemented them now according to this newer documentation: https://flask.palletsprojects.com/en/2.0.x/testing/, which is also using the test client